modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -36,7 +36,7 @@ If not set, the default value is based on `infrastructure.config.openshift.io/cl
 * AWS: `LoadBalancerService` (with external scope)
 * Azure: `LoadBalancerService` (with external scope)
 * GCP: `LoadBalancerService` (with external scope)
-* Other: `HostNetwork`.
+* Other: `HostNetwork` or `NodePort`
```

Comment:
I'm not sure if `NodePort` is interchangeable with `HostNetwork` or where it falls in the `endpointPublishingStrategy`. This is subject to change.